src/PrinterLog.php

```diff
@@ -93,6 +94,14 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
     public static function displayTabContentForItem(CommonGLPI $item, $tabnum = 1, $withtemplate = 0)
     {
         if ($item->getType() == Printer::getType() && $item->getID() > 0) {
+            // display the printer graph buttons component
+            TemplateRenderer::getInstance()->display('components/printer_graph_buttons.html.twig', [
+                'start_date' => $_GET['date_start'] ?? null,
```

Comment:
Not confirmed, but this directly let me think you may introduce XSS vulnerabilities.
Maybe `filter_var` can be used to validate the input